pkgs/test/pubspec.yaml

```diff
@@ -36,7 +36,7 @@ dependencies:
   stream_channel: ^2.1.0
 
   # Use an exact version until the test_api and test_core package are stable.
-  test_api: 0.7.5
+  test_api: 0.7.6
```

Comment:
@jakemac53 not sure what you want to do about versions here - these changes could be a `+1`, but it might be better to prepare them on a branch from the last release if you don't want to release any of the other changes pushed to `main` since the release.

Reply:
This is fine, we can release with the other changes as well.